components/PricingBlock/PricingBlock.tsx

```diff
@@ -19,6 +19,7 @@ import { IPricePlans } from "./types";
 import { VariantsTypography } from "../../enums/TypographyVariants";
 import { Typography } from "../../ui/Typography";
 import { SliderPricing } from "../SliderPricing/SliderPricing";
+import UnderlinedTitle from "../Underline/Underline";
```

Comment:
I suggest remove default exports from components because it difficult to debug and navigate in project i.e 
import UnderlinedTitle from "../Underline/Underline"; and import Bred from "../Underline/Underline";  are equal because export is default

Reply:
Also i suggest cut "dead" code. in Every imports you have ./<Component name>/<Component name> then i suggest transform it to ./<Component name>